src/cli/tc.rs

```diff
@@ -55,53 +56,30 @@ pub fn run() -> Result<()> {
 
 pub async fn init(file_path: &str) -> Result<()> {
```

Comment:
Can you add better documentation to the `init` command, currently it just says, that it initializes the current project.

Reply:
@tusharmath Sure, will do that in docs repo